content/en/network_performance_monitoring/installation.md

```diff
@@ -122,14 +127,51 @@ If you need to use Network Performance Monitoring on other systems with SELinux
 
 If these utilities do not exist in your distribution, follow the same procedure but using the utilities provided by your distribution instead.
 
+### Windows systems
+
+Data collection for Windows systems is now available in public beta for versions 2016 or later. Note that the driver currently monitors Windows hosts only, and not Windows containers. DNS metric collection is not currently supported for Windows systems.
```

Comment:
still wouldn't say `driver`.  

Note that network monitoring supports Windows hosts only, and not Windows containers.

Reply:
Updated!